docs/logs/discovery/tailing-log-files.md

```diff
@@ -0,0 +1,54 @@
+title: Shipping Log Files
+description: Shipping Log File Sources
+
+Enabling automatic logs shipping is accomplished by triggering the "Set up" button that opens the flyout for configuring log file shipping.
+
+![Setup Button](images/setup.png)
+
+In the flyout panel, you have to repeat the following steps to start shipping logs for the log file source:
+
+1. Enable the "Automatically ship <group-name> logs" toggle
+2. Select the destination Logs App. If the Logs App doesn't exist, you'll be able to create one from the same screen
+3. Click the `Save Changes` button
+
+The log shipper will automatically start tailing any log file that pertains to the service type for which the log shipping was set up.
+
+![Enable Log Shipping](images/logfile-flyout.png)
+
+## Defining glob patterns
+
+To narrow down which log files should be tailed, you can define include/exclude [glob](https://en.wikipedia.org/wiki/Glob_(programming)) patterns. The following glob wildcards are supported:
+
+- `*` matches any sequence of non-separator characters
+- `?` matches any single non-separator character
+- `[0-9]` matches character ranges
+
+When defining glob patterns, you can either specify include or exclude glob expressions. Exclude glob patterns take precedence over include patterns. This means, if there is a log file matched by both include and exclude glob patterns, it will results in the log file not being shipped.
+
+![Include and Exclude Glob Patterns](images/include-exclude-globs.png)
+
+As you're inputting the glob pattern, you'll be automatically getting the feedback on which log files are matched by include/exclude glob patterns.
+
+![Matched Hints](images/matched-globs.png)
+
+Going back to the main table, you'll notice two kind of visual indicators that signalize whether the particular log file source is included or discarded by the glob expression. By hovering over the `✓` or `✕` icons, you'll be able to figure out which exact glob pattern(s) are applied on the log file. If the log file is excluded from tailing, it is grayed out in the table row.
```

Comment:
two kind => two kinds
signalize => correct, though I think most people would just say "signal" - it's both a verb and a noun. :)
the particular => a particular
discarded => excluded